dev-client/src/theme.ts

```diff
@@ -300,7 +300,6 @@ export const theme = extendTheme({
     },
     Icon: {
       sizes: {
-        xs: 18,
```

Comment:
We actually will need `xs` for the button component work, so this needs to be restored.